src/trpc/router/restRouter.ts

```diff
@@ -111,4 +112,82 @@ export const restRouter = router({
             const actingUser = await userMethods.getByEmail(actingUserEmail);
             return bonusPointsMethods.change({ userId: targetUser.id, ...restInput }, actingUser);
         }),
+
+    getGroupById: restProcedure
+        .meta({
+            openapi: {
+                method: 'GET',
+                path: '/groups/info',
+            },
+        })
+        .input(
+            z.object({
+                id: z.string(),
+            }),
+        )
+        .output(
+            z.object({
+                id: z.string(),
+                name: z.string(),
+                description: z.string().nullable(),
+                supervisor: z
+                    .object({
+                        id: z.string(),
+                        name: z.string().nullable(),
+                        email: z.string(),
+                    })
+                    .nullish(),
+                memberships: z.array(
+                    z.object({
+                        user: z.object({
+                            id: z.string(),
+                            name: z.string().nullable(),
+                            email: z.string(),
+                        }),
+                        roles: z.array(
+                            z.object({
+                                name: z.string(),
+                            }),
+                        ),
+                    }),
+                ),
+                breadcrumbs: z.array(
+                    z.object({
+                        id: z.string(),
+                        name: z.string(),
+                    }),
+                ),
+            }),
+        )
+        .query(async ({ input }) => {
+            const group = await groupMethods.getById(input.id);
+            const memberships = await groupMethods.getMemberships(input.id);
+            const breadcrumbs = await groupMethods.getBreadcrumbs(input.id);
+            return { ...group, memberships, breadcrumbs };
+        }),
+
+    getGroupSuggestions: restProcedure
+        .meta({
+            openapi: {
+                method: 'GET',
+                path: '/groups/suggestions',
+            },
+        })
+        .input(
+            z.object({
+                query: z.string(),
```

Comment:
I would probably add limit and offset and have default max page

Reply:
done